source/token_api.rst

```diff
@@ -0,0 +1,119 @@
+************************************
+Token API
+************************************
+
+1. Get a Token
+=======================
+| URL endpoint: https://stamps.co.id/api/token/
+| Allowed method: POST
+| Requires authentication: 
+
+
+A. Request
+-----------------------------
+
+You can get a new token by calling the API with the these data
+
+
+=========================== =========== =======================
+Parameter                   Required    Description
+=========================== =========== =======================
+token                       Yes         Authentication string
+merchant                    Yes         Merchant User ID indicated which merchant will the user access
+=========================== =========== =======================
+
+
+Here's an example of an API call using cURL.
+
+.. code-block:: bash
+    
+    $ curl \
+    -X POST \
+    -H "Content-Type: application/json" \
+    -d '{"token": "23095sgtr95402mkdls954002", "merchant": 1}' \
+    https://stamps.co.id/api/token/
+
+B. Response
+-----------
+On a successful API call, Stamps will give you the accss_token that can be used to access our other APIs.
```

Comment:
`Stamps will give you a time limited JWT token that can be used to access our other APIs.`

Reply:
JWT di sini di-link aja ke `jwt.io`